src/IssueViz/IssueVizPackage.cs

```diff
@@ -33,7 +33,7 @@ namespace SonarLint.VisualStudio.IssueVisualization
     [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [Guid("4F3D7D24-648B-4F3B-ACB0-B83AFE239210")]
     [ProvideMenuResource("Menus.ctmenu", 1)]
-    [ProvideToolWindow(typeof(IssueVisualizationToolWindow), MultiInstances = false, Style = VsDockStyle.Float, Width = 325, Height = 400)]
+    [ProvideToolWindow(typeof(IssueVisualizationToolWindow), MultiInstances = false, Transient = true, Style = VsDockStyle.Float, Width = 325, Height = 400)]
```

Comment:
This is important: the window has to be Transient (= always closed when VS is opened). Otherwise, since the command is always visible ( = not attached to a context), it means that the user can manually open it and leave it opened when VS shuts down. It will then cause a goldbar to appear next time VS is opened, complaining about our window.